Library/TestHelpers/TestCase.swift

```diff
@@ -88,11 +88,11 @@ internal class TestCase: XCTestCase {
   }
 
   internal func preferredSimulatorCheck() {
-    let deviceName = ProcessInfo().environment["SIMULATOR_DEVICE_NAME"]
+    let deviceName = ProcessInfo().environment["SIMULATOR_VERSION_INFO"]
```

Comment:
@ifosli This was a funny catch. I always name my snapshot simulators something (this one is "New New Snapshot Simulator"). It looks like `SIMULATOR_DEVICE_NAME` would return "New New Snapshot Simulator" whereas `SIMULATOR_VERSION_INFO` is like `CoreSimulator 494.33 - Device: iPhone 8 Plus - Runtime: iOS 11.2 (15C107) - DeviceType: iPhone 8 Plus`.

Reply:
Makes sense! I didn't even know you could re-name simulators, so thanks for handling that case!